templates/dataset.html

```diff
@@ -13,6 +13,10 @@ <h3>Tags: {{ dataset.tags|join(", ") }}</h3>
 
     <p>{{ dataset.description }}</p>
 
+    {% if let Some(comment) = dataset.comment %} <p>Comment: {{ comment }}</p> {% endif %}
+
+    {% if let Some(last_checked) = dataset.last_checked %} <p>Last checked on source metadata portal: {{ last_checked }}</p> {% endif %}
```

Comment:
I think a simple `<p>Last checked: ...` would be preferable to avoid fixing the semantics too much, especially as we integrate more source which might have slightly different interpretations of this field.

Reply:
I choose this implementation to have a way to distinguish between fields that are empty strings or that are None.

Reply:
I don't mean the implementation on how to handle the `Option`, just the label that is part of the HTML, e.g.

```jinja
{% if let Some(last_checked) = dataset.last_checked %} <p>Last checked: {{ last_checked }}</p> {% endif %}
```

Reply:
OK, I'll go with that

Reply:
Looks fine, but now it is missing the colon ":" after "Last checked". :-)